testing/common_test_generator.py

```diff
@@ -11,7 +11,7 @@ def generate_serialized_trajectory(mdp, save_path):
     sparse_reward = 0
```

Comment:
I can verify this is still being used in overcoooked_test, so deleting it will result in case failure. Nathan put those tests together so maybe he should answer this questions..

Reply:
@nathan-miller23 do you know if we still use this file / where? My understanding is that we javascript doesn't even have encodings / dynamics anymore on their side, so I think it can be deleted – but wanted to double check with you